main.go

```diff
@@ -39,6 +40,11 @@ type GAE struct {
 	// FlexImage tells the plugin where to pull the image from when deploying a Flexible
 	// VM instance. Example value: 'gcr.io/nyt-games-dev/puzzles-sub:$COMMIT'
 	FlexImage string `json:"flex_image"`
+	// TemplateVars allows users to pass a set of key/values to be injected into the
+	// various yaml configuration files. To use, the keys in this map must be referenced
+	// in the yaml files with Go's templating syntax. For example, the key "ABC" would be
+	// referenced with {{ .ABC }}.
+	TemplateVars map[string]interface{}
```

Comment:
Do you need a `json:"template_vars"`?

Also I think in the config this should just be `vars:`.

This is super reminiscent of drone-gke.

Reply:
good catch! adding.